app/forms/work_package_types/subject_configuration_form.rb

```diff
@@ -84,17 +86,9 @@ def validation_message_for(attribute)
     end
 
     def pattern_input_caption
-      I18n.t(
-        "types.edit.subject_configuration.pattern.caption_with_supported_attributes_link",
-        link: make_link(
-          ::OpenProject::Static::Links.url_for(:enterprise_features, :work_package_subject_generation),
-          I18n.t("types.edit.subject_configuration.pattern.supported_attributes_link")
-        )
-      ).html_safe
-    end
-
-    def make_link(href, link_text)
-      render(Primer::Beta::Link.new(href:, target: "_blank", underline: true)) { link_text }
```

Comment:
Note: This was previously rendered with `underline: true`. In the helper I don't specify `underline` at all, but the link is rendered underlined anyways (I believe this is due to changes to the default styling of links).